test/functional/Java11andUp/src/org/openj9/test/condy/CondyGenerator.java

```diff
@@ -170,6 +171,47 @@
 			mv.visitEnd();
 		}
 
+		{
+			mv = cw.visitMethod( ACC_PUBLIC | ACC_STATIC, "triggerGC", "()I", null, null );
```

Comment:
Does this have to be a generated method or can it live on GarbageCollectionCondyTest.java ?

Reply:
I will test moving this in to the test file